src/pages/PassportRedemption/TrackScreen.tsx

```diff
@@ -156,40 +168,42 @@ const Track = ({ setCurrentScreenView }: Props) => {
             )}
           </Column>
 
-          <Row>
-            <Label htmlFor="instagram-handle">
-              Instagram Handle (for Digital Giveaway)
-            </Label>
-            <SupporterTooltip
-              title={
-                <React.Fragment>
-                  <ToolTipTable>
-                    <tbody>
-                      <tr>
-                        To be entered into our weekly Digital Giveaways, visit 3
-                        merchants and post and tag <b>@sendchinatownlove</b>{' '}
-                        with your food crawl pictures on Instagram. Enter your
-                        Instagram handle so we can track your entries.
-                      </tr>
-                    </tbody>
-                  </ToolTipTable>
-                </React.Fragment>
-              }
-              enterTouchDelay={10}
-              placement="left"
-            >
-              <div>
-                <img src={CrawlInfoIcon} alt="instagram-crawl-info" />
-              </div>
-            </SupporterTooltip>
-          </Row>
-          <InputField
-            name="instagram-handle"
-            type="text"
-            onChange={(e) => setinstagramHandle(e.target.value)}
-            value={instagramHandle}
-            placeholder="@"
-          />
+          <Column>
+            <Row className="row-no-margin">
+              <Label htmlFor="instagram-handle">
+                Instagram Handle (for Digital Giveaway)
+              </Label>
+              <SupporterTooltip
+                title={
+                  <React.Fragment>
+                    <ToolTipTable>
+                      <tbody>
+                        <tr>
+                          To be entered into our weekly Digital Giveaways, visit 3
+                          merchants and post and tag <b>@sendchinatownlove</b>
```

Comment:
When I was doing the FAQ markup, I read that `<strong>` or CSS font-weight property is preferred over `<b>` for styling text??